spec/views/content_items/show.html.erb_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'content_items/show.html.erb', type: :view do
+  let(:content_item) { build(:content_item) }
+
+  it 'renders the title of the content item' do
+    content_item.title = 'A Title'
+    assign(:content_item, content_item)
+    render
+
+    expect(rendered).to have_selector('h1', text: 'A Title')
+  end
+
```

Comment:
Oh. I'll see what I can do. I might just tack on a commit to do this at the end, rather rebasing again.